_lifeofinternatpuppet/post_16.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Week 16
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 16
+---
+
+Hello Everyone!
+
+Not too long to Christmas now, of course?
+
+This has been the busiest week of my placement so far, before the team wraps up for Christmas holidays!
+
+Some of the key highlights included:-
+
+1. Module Release Prep
+      *[puppetlabs-motd](https://tickets.puppetlabs.com/browse/IAC-1308) (v4.3.0)
+      *[puppetlabs-tomcat](https://tickets.puppetlabs.com/browse/IAC-1330)
+          (https://github.com/puppetlabs/puppetlabs-tomcat/pull/426) (v4.3.0)
+      *[puppetlabs-tagmail](https://tickets.puppetlabs.com/browse/IAC-1349)
+          (https://github.com/puppetlabs/puppetlabs-tagmail/pull/196)
```

Comment:
If you formatted like this:
```suggestion
**Module Release Prep:**
- [puppetlabs-motd (IAC-1308)](https://tickets.puppetlabs.com/browse/IAC-1308) (v4.3.0)
- [puppetlabs-tomcat (IAC-1330)](https://tickets.puppetlabs.com/browse/IAC-1330)
- [puppetlabs-tomcat (PR #426)](https://github.com/puppetlabs/puppetlabs-tomcat/pull/426) (v4.3.0)
- [puppetlabs-tagmail (IAC-1349)](https://tickets.puppetlabs.com/browse/IAC-1349)
- [puppetlabs-tagmail (PR #196)](https://github.com/puppetlabs/puppetlabs-tagmail/pull/196)
```

You'd get this in the browser:
![image](https://user-images.githubusercontent.com/55992494/102493610-5368fd80-406b-11eb-863e-aee42929a41e.png)